src/Keywords/KeywordsInterface.php

```diff
@@ -18,13 +18,17 @@
  * @phpstan-type TLanguage string
  * @phpstan-type TKeywordsArray array{feature: string, background: string, scenario: string, scenario_outline: string, examples: string, given: string, when: string, then: string, and: string, but: string}
  * @phpstan-type TMultiLanguageKeywords array<TLanguage, TKeywordsArray>
+ * @phpstan-type TKeywordsString string
+ * @phpstan-type TStepKeywordsType 'Given'|'When'|'Then'|'And'|'But'
```

Comment:
As I understand, these are non-translatable keyword "identifiers" that are later on mapped to translations. If that is the case, then this should be fine.